source/about/cloud-dedicated.rst

```diff
@@ -0,0 +1,114 @@
+- SRTP (mandatory): Used to encrypt all media packets (i.e. those containing voice or screen share). Supports ``AEAD_AES_128_GCM`` and ``AES128_CM_HMAC_SHA1_80`` algorithms.
+
+Cloud native exports
+~~~~~~~~~~~~~~~~~~~~
+
+Mattermost supports optional :ref:`filestore configuration settings <configure/experimental-configuration-settings:enable dedicated export filestore target>` to direct compliance and bulk export data to a separate S3 bucket from standard files. This separate bucket can be configured to allow for secure access by Mattermost Cloud teams as well as the workspace admins who manage a given installation. The exports can also be accessed by generating unique download links as needed.
+
+The following diagram provides a high-level view of how this functionality works:
+
+.. image:: ../images/mattermost-cloud-native-export-architecture-diagram.png
+  :alt: An architecture diagram showing a high-level view of how Mattermost Cloud Native exports works.
+
+SMTP
+~~~~
+
+Email sent from Mattermost Cloud Dedicated uses SendGrid, and the connection to SendGrid is encrypted.
+
+Audit and observability
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Cloud Dedicated provides access to :doc:`audit and system logs </manage/logging>` generated by the application. 
+
+Bring your own cloud
+~~~~~~~~~~~~~~~~~~~~~
+
+With Bring Your Own Cloud (BYOC), use your own cloud infrastructure instead of using the Mattermost-managed infrastructure to manage your Mattermost instance through Mattermost command and control while keeping your data in your own cloud.
+
+Consider using BYOC if:
+
+- You use Amazon Web Service (AWS) or Azure.
+- You're an Enterprise customer with a committed deal with Mattermost.
+- You have strict regulatory requirements or special compliance requirements, BYOC may will be the best option for you.
+- You require the visibility of all traffic within any VPC you operate in, or need frequent auditing capabilities.
+- You're looking for fine-grained network control. BYOC only requires specific network access (non-internet accessible) for Mattermost (for example, service management or troubleshooting) to deploy and manage Mattermost instance. This allows you to customize your network to meet any internal requirements or requirements of your customers.
+- Take advantage of cost savings plans, committed use discounts, or other strategies to save on compute and storage infrastructure costs related to your Mattermost instance.
+
+Contact a `Mattermost Expert <https://mattermost.com/contact-sales/>`_ to discuss this option in more detail.
+
+Customization
+~~~~~~~~~~~~~~
+
+Approved plugins developed and/or tested by Mattermost are supported and available in the `Mattermost Marketplace <https://mattermost.com/marketplace/>`_. Custom plugins and integrations outside of Mattermost Marketplace aren’t currently supported.
```

Comment:
Do we want to link to the KB article https://support.mattermost.com/hc/en-us/articles/5346624843924-Mattermost-Plugin-Marketplace-List-as-of-October-9th-2023 for more guidance on what is supported?

Reply:
No, because that list of supported plugins in the forum post is no longer accurate.